py3status/modules/sysdata.py

```diff
@@ -9,10 +9,8 @@
         '[\?color=mem Mem: {mem_used}/{mem_total} GB ({mem_used_percent}%)]')*
     mem_unit: the unit of memory to use in report, case insensitive.
         ['dynamic', 'KiB', 'MiB', 'GiB'] (default 'GiB')
-    padding: length of space padding to use on the left
-        (default 0)
-    precision: precision of values
-        (default 2)
+    temp_unit: unit used for measuring the temperature
+        (default '°C')
```

Comment:
@tobes I added the option for Fahrenheit and Kelvin
`C` or `°C` gives Celsius
`F` or `°F` gives Fahrenheit
`K` gives Kelvin (Celsius + 273.15)